web-console/Storage/OCS/OCS _Installation_Workflow.md

```diff
@@ -0,0 +1,98 @@
+# OCS Install Workflow
+
+### Operator Hub
+
+There will be 2 OCS operator entries in Operator Hub covering each of the following modes. 
+* Converged Mode -- Local Cluster / Hyperconverged 
+* Independent Mode -- External Cluster (Ceph Cluster is deployed externally)
+
+### Converged Mode Design
+
+Operator Hub
+![Operator Hub operators view](img/OCS-operator-view.png)
+
+Installation
+![Operator Hub operators view](img/OCS-Install-step1.png)
+
+![Operator Hub operators view](img/OCS-Install-step2.png)
+
+After the subscription process, the user will see the subscribed operator page. 
+In the installed operator view the user can see the OCS operator installed. 
+
+![Operator Hub operators view](img/Installed_operators_list.png)
+
+
+# OCS Installation UPI (deployed on the cloud)
+
+
+OCS Overview page
+![OCS-View](img/OCS-View.png)
+
+Clicking on "create new" (OCS Service) Will start the following installation flow: 
+### Step 1- Select Nodes
+ * Admin needs to select 3 nodes to label with “node-role.kuberentes.io/storage=" "node.ocs.openshift.io/cluster=" to be used for the OCS cluster.
+ * For each node selected, all available devices will be selected on the node by default.  Admin can elect to change the device selection, i.e. to use a subset of the devices if he/she needs to.
+
+![Installation page](img/Create_new_OCS_01.png)
+
+* Clicking on the devices button link will open  the “select Device” modal. the user is able to choose which devices he wants to use to support the OCS service.
```

Comment:
@yuvalgalanti @shirimordechay 

TODOs missing from the design:
- We still need to provide the filter for the device table.
- We also need to provide the "device" modal for when it's AWS and VMware.